TarSCM/scm/git.py

```diff
@@ -129,24 +129,37 @@ def fetch_submodules(self):
     def update_cache(self):
         """Update sources via git."""
         # Force origin to the wanted URL in case it switched
```

Comment:
catch exeption
* in case of server context
  * remove cache dir
  * run "fetch_upstream_scm"
* else
  * log info to cleanup manually
  * raise SystemExit